code/src/java/gmgen/plugin/PcgCombatant.java

```diff
@@ -1041,9 +1061,12 @@ protected String getStatBlockLineSpells()
 			return statBuf.toString();
 		}
 
-		protected void statBlockLineSpellBook(PlayerCharacter aPC, StringBuilder statBuf, Collection<PObject> classList, String spellBookName)
+		private void statBlockLineSpellBook(PlayerCharacter aPC,
+		                                    StringBuilder statBuf,
+		                                    Collection<PObject> classList,
+		                                    String spellBookName)
 		{
-			Set<PObject> classes = new HashSet<>(classList);
+			Iterable<PObject> classes = new HashSet<>(classList);
```

Comment:
I still don't see the point of this - why do we not want to treat this as a Set?
I'm actually getting annoyed at some of these because they are making me switch them BACK when I want to use something the "proper" way.
So basically, beyond "Set" and "List" I believe these have negative value.